distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -693,7 +693,7 @@ private static Integer findPortNumber(Integer portToTry) {
     return options(
         editConfigurationFilePut(SYSTEM_PROPERTIES_REL_PATH, "solr.client", "HttpSolrClient"),
         editConfigurationFilePut(
-            SYSTEM_PROPERTIES_REL_PATH, "solr.http.url", "http://localhost:8994/solr"),
+            SYSTEM_PROPERTIES_REL_PATH, "solr.http.url", "http://localhost:9994/solr"),
```

Comment:
don't forget to add a line in README.md with this info

```
## Accessing Solr
Solr runs on port 9994 during the itests
```

Reply:
Separate PR

Reply:
https://github.com/rzwiefel/ddf-solr/pull/7